cmpserver/server.go

```diff
@@ -61,6 +64,11 @@ func NewServer(initConstants plugin.CMPServerInitConstants) (*ArgoCDCMPServer, e
 		grpc.StreamInterceptor(grpc_middleware.ChainStreamServer(streamInterceptors...)),
 		grpc.MaxRecvMsgSize(apiclient.MaxGRPCMessageSize),
 		grpc.MaxSendMsgSize(apiclient.MaxGRPCMessageSize),
+		grpc.KeepaliveEnforcementPolicy(
+			keepalive.EnforcementPolicy{
+				MinTime: (time.Duration(10) * time.Second),
+			},
+		),
```

Comment:
Would this be a good candidate for a common.go constant?

Reply:
Yup, I believe so - especially since it needs to match in multiple places.